2018/05.md

```diff
@@ -60,7 +60,7 @@ Deadline for advancement eligibility: [**12th May 2018, 10:00 EDT**](https://www
     1. Such proposals *should* include supporting materials when possible.
 1. Proposals looking to advance to stages 2, 3, or 4 must be added (and noted as such) *along with the necessary materials* prior to the deadline.
     1. Such proposals *must* link to supporting materials prior to the deadline, or they will not be eligible for advancement. If these materials change substantially after the deadline, the proposal may be disqualified from advancement, based on the committee’s judgment.
-1. Timeboxed topics may be 15, 30, 45, or 60 minutes in length.
+1. Proposal-based agenda items should be sorted primarily by stage (descending), secondarily by timebox (ascending), and finally by insertion date.
```

Comment:
Should the 4 times still be listed? We don’t want people price-is-righting their timebox times :-)

Reply:
I wouldn't mind a bit of price-is-righting. Whatever encourages shorter timeboxes.